cli/src/semgrep/commands/ci.py

```diff
@@ -82,12 +84,20 @@ def yield_valid_patterns(patterns: Iterable[str]) -> Iterable[str]:
         yield pattern
 
 
-def yield_exclude_paths(requested_patterns: Sequence[str]) -> Iterable[str]:
-    patterns = [*yield_valid_patterns(requested_patterns), *ALWAYS_EXCLUDE_PATTERNS]
```

Comment:
This call was eliminated, but the function wasn't deleted, and the functionality not replaced. Was this not being used? Was this intentional?

Reply:
If this was an omission please add a test to make sure it doesn't get removed again.

Reply:
`get_exclude_paths` replaces the functionality; renamed since it is no longer a generator.